### ISSUE TITLE
Issue about saving checkpoint

### ISSUE THREAD

**Issue by CheungKaKei** [njucckevin/SeeClick#46]
Thank you for your wonderful work!

I encountered some problems during fine-tuning mind2web data using the SeeClick ckpt you released.  The problem occurs when I try to test the fine-tuned mind2web data as follows: 

1) The ckpt saved after fine-tuning on mind2web is significantly smaller than the version you released (only about 400MB), which it's obviously abnormal.

2) Then the test.py can not correctly find the ckpt. Function AutoModelForCausualLM actually seems to read another previously stored path, making the undesired testing. 

In summary, it is suspected that the checkpoint is somehow stored incorrectly or incompletely.  Although I do my best to follow all the options you stated on this GitHub page, I still probably made some incorrect saving settings by mistake. I would really appreciate it if you could also give me some advice.

the abnormal ckpt I get:
![the abnormal ckpt](https://github.com/user-attachments/assets/9a0f92bb-264d-482d-b57d-4c8bfba4c567)

the running cmd I used:
`
export NCCL_IB_DISABLE=1; export NCCL_P2P_DISABLE=1; bash finetune/finetune_lora_ds.sh --save-name TTTT --max-length 704 --micro-batch-size 8 --save-interval 500 --train-epochs 2 --nproc-per-node 2 --data-path ./Test_Env/mind2web/mind2web_annots/mind2web_train_sft.json --learning-rate 3e-5 --gradient-accumulation-steps 8 --qwen-ckpt ./Qwen-VL-Chat --pretrain-ckpt ./SeeClick-pretrain/ --save-path ./Test_Env/mind2web/
`

the options in finetune_lora_ds.sh:
![setting](https://github.com/user-attachments/assets/7f649eed-84b1-4e86-a580-5bd4e4006b21)

**Comment by CheungKaKei** [comment]
Fixed! It seems I misunderstood the lora and base model ckpt, thanks!
